lib/ansible/modules/cloud/vmware/vcenter_folder.py

```diff
@@ -181,44 +181,48 @@ def ensure(self):
                                                    folder_type=folder_type,
                                                    parent_folder=p_folder_obj)
                 if child_folder_obj:
-                    results['result'] = "Folder %s already exists under" \
-                                        " parent folder %s" % (folder_name, parent_folder)
+                    results['path'] = self.get_folder_path(child_folder_obj)
```

Comment:
Add a sample example in RETURN documentation for this change.

Reply:
ok